layers/spell-checking/packages.el

```diff
@@ -31,14 +34,12 @@
         :documentation
         "Enable automatic spell checking."
         :evil-leader "tS")
+
       (evil-leader/set-key
         "Sd" 'ispell-change-dictionary
         "Sn" 'flyspell-goto-next-error))
     :config
-    (progn
-      (when spell-checking-enable-by-default
-        (flyspell-prog-mode))
```

Comment:
I'm not sure what _this_ was supposed to do… `flyspell-prog-mode` is a wrapper around `flyspell-mode`, i.e. effectively a buffer local minor mode—calling it this way has no effect at all.